{
  "debianbug": 448838,
  "description": "The internationalization (i18n) framework in Django 0.91, 0.95, 0.95.1, and 0.96, and as used in other products such as PyLucid, when the USE_I18N option and the i18n component are enabled, allows remote attackers to cause a denial of service (memory consumption) via many HTTP requests with large Accept-Language headers.",
  "releases": {
    "bullseye": {
      "fixed_version": "0.96-1.1",
      "repositories": {
        "bullseye": "1:1.11.23-1~deb10u1"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "buster": {
      "fixed_version": "0.96-1.1",
      "repositories": {
        "buster": "1:1.11.21-1",
        "buster-security": "1:1.11.23-1~deb10u1"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "jessie": {
      "fixed_version": "0.96-1.1",
      "repositories": {
        "jessie": "1.7.11-1+deb8u3",
        "jessie-security": "1.7.11-1+deb8u7"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "sid": {
      "fixed_version": "0.96-1.1",
      "repositories": {
        "sid": "2:2.2.4-1"
      },
      "status": "resolved",
      "urgency": "low"
    },
    "stretch": {
      "fixed_version": "0.96-1.1",
      "repositories": {
        "stretch": "1:1.10.7-2+deb9u4",
        "stretch-security": "1:1.10.7-2+deb9u6"
      },
      "status": "resolved",
      "urgency": "low"
    }
  },
  "scope": "remote"
}